common/tests/conftest.py

```diff
@@ -108,27 +111,27 @@ def guardian_api_client():
 @pytest.fixture()
 def project_user_api_client(project):
     user = UserFactory()
-    assign_perm("admin", user, project)
+    assign_perm(ProjectPermission.ADMIN.value, user, project)
     return create_api_client_with_user(user)
 
 
 @pytest.fixture(params=(False, True), ids=("object_perm", "model_perm"))
 def publisher_api_client(request, project):
     user = UserFactory()
-    assign_perm("admin", user, project)
+    assign_perm(ProjectPermission.ADMIN.value, user, project)
 
     if request.param:
-        assign_perm("publish", user, project)
+        assign_perm(ProjectPermission.PUBLISH.value, user, project)
     else:
-        assign_perm("projects.publish", user)
+        assign_perm(f"projects.{ProjectPermission.PUBLISH.value}", user)
```

Comment:
Would it be better to import `PERM_CAN_PUBLISH_EVENTS` from projects.models and use it here?
> PERM_CAN_PUBLISH_EVENTS = f"projects.{ProjectPermission.PUBLISH.value}"